sdk/storage/AzureStorageBlob/Source/Operation/MultiBlobOperation.swift

```diff
@@ -41,7 +41,7 @@ internal class MultiBlobOperation: TransferOperation {
 
     // MARK: Public Methods
 
-    override public func main() {
```

Comment:
Looks like a regression from #306, try updating your SwiftLint + SwiftFormat.

Going forward I wonder if there is any way to better protect about rule changes between versions of these tools. Like maybe have a bot that applies the latest version of swiftlint + swiftformat changes and submits PRs automatically.

Reply:
I would hesitate to do that because, at least drawing my experience in Python, updates to pylint would come out that we absolutely did not want, and we would often pin to an older, specific version of pylint.

Reply:
Hmm, I wonder if there is a way we can pin the tools versions for swiftlint / swiftformat. At the very least we *could* have the build step that runs them download (or use existing copy) of a specific version of the tool rather than relying on devs to have their own system-wide copies acquired from e.g. Homebrew